example/example/settings.py

```diff
@@ -13,6 +13,7 @@
 from pathlib import Path
 
 from spid_config.spid_settings import *
+from spid_config.spid_settings_docker import *
```

Comment:
usiamo solo spid_settings, unufichiamo i file, evitiamo repliche as possibile (nonostate tutto)

Reply:
il file di per i settaggi di Docker fa solo l'override dei parametri strettamente necessari, il resto lo lascia invariato. Se hai un'idea precisa di come fare ci sentiamo in VC?

Reply:
la demo deve essere as-is sia in python venv che in docker, altrimenti ogni commit si andrà a sostituire al precedente.
Qua basta creare dei getattr con fallback, vedi un po se ti viene in mente qualcosa altrimenti ci penso io

Reply:
Ho fatto le modifiche, risolvo la conversazione che ormai è "Outdated". Se c'è altro segnalamelo pure nel codice aggiornato.